integration

```diff
@@ -781,6 +781,7 @@
   "jeroenterheerdt/HAsmartirrigation",
   "jesserockz/ha-leafspy",
   "Jezza34000/homeassistant_petkit",
+  "Jezza34000/homeassistant_veolia",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>